CHANGELOG.md

```diff
@@ -22,7 +22,10 @@ All notable changes to this project will be documented in this file.
 
 ## [1.7.2] - 2018-04-30
 
+- Add support for private repositories using docker credential stores/helpers (fixes [\#567](https://github.com/testcontainers/testcontainers-java/issues/567))
```

Comment:
Please could you shift this up to the `UNRELEASED` section?

Also, it looks like this same line appears (more or less the same) three times - merge issue?

I think that for now we should draw a line in the sand and release this for Mac and Linux - could we say so in the changelog entry?